desktop/src/main/kotlin/com/github/picture2pc/desktop/viewmodel/mainscreen/PictureDisplayViewModel.kt

```diff
@@ -61,9 +61,16 @@ class PictureDisplayViewModel(
         this.displayPictureSize = displayPictureSize
     }
 
+    fun rotate(clockwise: Boolean) {
+        pP.rotate(clockwise)
+    }
+
     fun reset() {
         mHVM.clear()
-        setPicture(pictures.replayCache[selectedPictureIndex.value])
```

Comment:
could have been done like `setPicture(pictures.replayCache[selectedPictureIndex.value]  ?: return)`. Just for future